### PR TITLE
{lib}[intel/2017a] pandas v0.21.0

### PR DESCRIPTION
**Pull request by wpoely86**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic200 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E7-8891 v4 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/97c84810b5aaa2237c043f6082af38a3 for a full test report.

File: easybuild/easyconfigs/p/pandas/pandas-0.21.0-intel-2017a-Python-2.7.13.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "https://pypi.python.org/pypi/pandas/"
+description = """pandas is an open source, BSD-licensed library providing high-performance, easy-to-use data structures
+ and data analysis tools for the Python programming language."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
```
Comment (boegel) [inline easybuild/easyconfigs/p/pandas/pandas-0.21.0-intel-2017a-Python-2.7.13.eb]: @wpoely86 Please include a SHA256 checksum? (use `eb --inject-checksums`)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2548.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7ff78e3adf9b90fbc689db6c6091fb2a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2131.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6f4bc96dcb44793f49907079e2971f03 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**